.github/workflows/cherrypick-to-beta.yaml

```diff
@@ -0,0 +1,124 @@
+# Copyright 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+name: Cherry-pick Labeled PR to Beta Branch
```

Comment:
Can you send a design doc for this feature?

Reply:
Sure working on the doc